Update Set-CsOnlineLisSwitch.md

danny-levin opened the pull request.

opbld31 commented:
Docs Build status updates of commit _[f8d6a5b](https://github.com/danny-levin/office-docs-powershell/commits/f8d6a5ba3e8adbc903e544132934ea2cd940ca4f)_: 

:clock10: **Full build**: the average full build time is _16 min(s) 43 sec(s)_, this is based on the last _1000_ build(s) for this repository.

opbld33 commented:
Docs Build status updates of commit _[f8d6a5b](https://github.com/danny-levin/office-docs-powershell/commits/f8d6a5ba3e8adbc903e544132934ea2cd940ca4f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[skype/skype-ps/skype/Set-CsOnlineLisSwitch.md](https://github.com/danny-levin/office-docs-powershell/blob/patch-4/skype/skype-ps/skype/Set-CsOnlineLisSwitch.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/set-csonlinelisswitch?view=skype-ps&branch=pr-en-us-5545) (skype-ps) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C5%5C12%5C3603fd28-efce-f87f-e986-fc8a694fc312%5CPullRequest%5C202005121436015107-5545%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=vwTYAa7axIGzTbosFyULJDEGqH4PVoBsEpz2x5wyS0c%3D&st=2020-05-12T14%3A57%3A52Z&se=2020-06-12T15%3A02%3A52Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

opbld33 commented:
Docs Build status updates of commit _[b91a018](https://github.com/danny-levin/office-docs-powershell/commits/b91a0187e799a54b642e55600d3c67d52008fc47)_: 

:clock10: **Full build**: the average full build time is _16 min(s) 46 sec(s)_, this is based on the last _1000_ build(s) for this repository.

opbld32 commented:
Docs Build status updates of commit _[b91a018](https://github.com/danny-levin/office-docs-powershell/commits/b91a0187e799a54b642e55600d3c67d52008fc47)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[skype/skype-ps/skype/Set-CsOnlineLisSwitch.md](https://github.com/danny-levin/office-docs-powershell/blob/patch-4/skype/skype-ps/skype/Set-CsOnlineLisSwitch.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/set-csonlinelisswitch?view=skype-ps&branch=pr-en-us-5545) (skype-ps) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C5%5C12%5C3603fd28-efce-f87f-e986-fc8a694fc312%5CPullRequest%5C202005121652537504-5545%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=zZEe%2FQmkHIuKzfkVSGPo9lLCAG88pJWN0TP6YQY6MMg%3D&st=2020-05-12T17%3A11%3A36Z&se=2020-06-12T17%3A16%3A36Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

opbld30 commented:
Docs Build status updates of commit _[0fe01df](https://github.com/danny-levin/office-docs-powershell/commits/0fe01df4af74bf8ded431986f0503edfc86735be)_: 

:clock10: **Full build**: the average full build time is _16 min(s) 54 sec(s)_, this is based on the last _1000_ build(s) for this repository.

kenwith commented:
I think this was already merged in the previous PR. Let me know if that isn't correct and more updates still needed.

opbld32 commented:
Docs Build status updates of commit _[0fe01df](https://github.com/danny-levin/office-docs-powershell/commits/0fe01df4af74bf8ded431986f0503edfc86735be)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[skype/skype-ps/skype/Set-CsOnlineLisSwitch.md](https://github.com/danny-levin/office-docs-powershell/blob/patch-4/skype/skype-ps/skype/Set-CsOnlineLisSwitch.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/set-csonlinelisswitch?view=skype-ps&branch=pr-en-us-5545) (skype-ps) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C5%5C13%5C3603fd28-efce-f87f-e986-fc8a694fc312%5CPullRequest%5C202005130017215118-5545%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=L8654C5hn%2FA9dzKNbyWHGhYFB5bvO0hpp9p4N5QcA3I%3D&st=2020-05-13T00%3A34%3A59Z&se=2020-06-13T00%3A39%3A59Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

danny-levin commented:
Thx, it looks good.
(I changed a description of 2 parameters in 2 cmdlets (set switch & set port) and wanted both to include exactly the same description. Initially, it was not the case, hence I updated it again… - sorry about that…)

From: Ken Withee <notifications@github.com>
Sent: Wednesday, May 13, 2020 3:20 AM
To: MicrosoftDocs/office-docs-powershell <office-docs-powershell@noreply.github.com>
Cc: Danny Levin <dannyle@microsoft.com>; Author <author@noreply.github.com>
Subject: Re: [MicrosoftDocs/office-docs-powershell] Update Set-CsOnlineLisSwitch.md (#5545)


I think this was already merged in the previous PR. Let me know if that isn't correct and more updates still needed.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Foffice-docs-powershell%2Fpull%2F5545%23issuecomment-627668555&data=02%7C01%7Cdannyle%40microsoft.com%7C6caf6c5f15fe403fd4d308d7f6d36a17%7C72f988bf86f141af91ab2d7cd011db47%7C0%7C0%7C637249260187186290&sdata=mXKeYpy7tG%2FpLRe6pCavX8uFQBtV5aW%2Fct1PA0E6PRg%3D&reserved=0>, or unsubscribe<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAJLYUC6NIB7AAITNDH2QVFDRRHRT7ANCNFSM4M64HD5Q&data=02%7C01%7Cdannyle%40microsoft.com%7C6caf6c5f15fe403fd4d308d7f6d36a17%7C72f988bf86f141af91ab2d7cd011db47%7C0%7C0%7C637249260187196289&sdata=uSlIcw2HQolvU277GHaeFb8J5B77fIzQckuLVmLClIA%3D&reserved=0>.

opbld32 commented:
Docs Build status updates of commit _[0fe01df](https://github.com/danny-levin/office-docs-powershell/commits/0fe01df4af74bf8ded431986f0503edfc86735be)_: 

:clock10: **Full build**: the average full build time is _16 min(s) 54 sec(s)_, this is based on the last _1000_ build(s) for this repository.

opbld33 commented:
Docs Build status updates of commit _[0fe01df](https://github.com/danny-levin/office-docs-powershell/commits/0fe01df4af74bf8ded431986f0503edfc86735be)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[skype/skype-ps/skype/Set-CsOnlineLisSwitch.md](https://github.com/danny-levin/office-docs-powershell/blob/patch-4/skype/skype-ps/skype/Set-CsOnlineLisSwitch.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/skype/set-csonlinelisswitch?view=skype-ps&branch=pr-en-us-5545) (skype-ps) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C5%5C13%5C3603fd28-efce-f87f-e986-fc8a694fc312%5CPullRequest%5C202005130417180095-5545%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=%2BDhyFUGCL5D%2FEzgscRGLHWLrU%2FmQsqj9eBcNgKvV5gw%3D&st=2020-05-13T04%3A35%3A50Z&se=2020-06-13T04%3A40%3A50Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.
